core/src/main/scala/pureconfig/error/ConfigReaderFailure.scala

```diff
@@ -58,24 +58,19 @@ object ConfigValueLocation {
 
 /**
  * A representation of a failure that might be raised from reading a
- * ConfigValue. The failure contains an optional location of the ConfigValue
- * that raised the error.
+ * ConfigValue. The failure contains an optional physical location of the
```

Comment:
Minor suggestion: "physical location" is a bit strange. Consider changing to "file system location".